src/source/feed-details.rst

```diff
@@ -0,0 +1,1443 @@
+.. _feed-details:
+
+Feed Fields
+===========
+
+A set of required and optional fields defined by feed is listed below.
+
+====================================== ==================================== ===================  =========== 
+Field                                  Description                          Restrictions         Mandatory 
+====================================== ==================================== ===================  =========== 
+:ref:`feed_vendorId`                   **unique** ad identifier             max. 64 chars        yes
+:ref:`feed_externalId`                 **deprecated**                       --                   --
+:ref:`feed_sellerName`                 your company name                          max. 60 chars        no
+:ref:`feed_t`                          product title                        see :ref:`feed_t`    yes
+:ref:`feed_descr`                      product description                  :ref:`feed_descr`    yes       
+:ref:`feed_categoryId`                 category identifier                  numeric, positive    yes       
+:ref:`feed_status`                     desired status (default ACTIVE)      ACTIVE,PAUSED        no       
+:ref:`feed_url`                        product URL                          max. 2048 chars      no        
+:ref:`feed_vanityUrl`                  displayed URL                        max. 256 chars       no        
+:ref:`feed_priceType`                  sales model for product              enum                 yes       
+:ref:`feed_price`                      product price in cents if applicable positive integer     yes/no       
+:ref:`feed_originalPrice`              original price before discount       positive integer     no        
+\* :ref:`feed_media`                   product images                       :ref:`feed_media`    no
+\* :ref:`feed_attr`                    collection of product attributes     :ref:`feed_attr`     no        
+\* :ref:`feed_budget`                  budget details                       :ref:`feed_budget`   no        
```

Comment:
what's the `\*` for?

Reply:
hm, I see the problem. I'd make 2 separate tables for TSV and XML, which can refer to same explanation for most of the stuff, but could also point to separate explanation where TSV and XML differ. This way the 1st column of this table can keep the name as is in the TSV or XML, and for explanation you use the reference.
What do you think?